<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Variables</class>
 <widget class="QMainWindow" name="Variables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_parametros">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Parametros </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Diseño del modelo de costos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_infocliente">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Información del cliente </string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_salir">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>560</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR </string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Modelo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_agregar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar producto </string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Oferta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Modificar Modelo</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>551</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
